.vscode/settings.json

```diff
@@ -16,5 +18,23 @@
     ],
     "python.analysis.extraPaths": [
         "/vendor"
-    ]
-}
+    ],
+    "files.associations": {
+        "*.html": "jinja-html"
```

Comment:
Do we need to add this extension to the devcontainer config?

Reply:
Added in devcontainer